wpimath/src/main/java/edu/wpi/first/math/controller/SimpleMotorFeedforward.java

```diff
@@ -175,57 +192,57 @@ public <U extends Unit> Voltage calculate(
     if (ka == 0.0) {
       // Given the following discrete feedforward model
       //
-      //   uₖ = B_d⁺(rₖ₊₁ − A_d rₖ)
+      // uₖ = B_d⁺(rₖ₊₁ − A_d rₖ)
       //
       // where
       //
-      //   A_d = eᴬᵀ
-      //   B_d = A⁻¹(eᴬᵀ - I)B
-      //   A = −kᵥ/kₐ
-      //   B = 1/kₐ
+      // A_d = eᴬᵀ
```

Comment:
Are these from a different PR? The original indentation was fine.

Reply:
This was spawned from the main branch.

Reply:
It looks like the formatter "fixed" it

Reply:
Do you want me to go ahead with fixing the indents.